test/e2e/config/default/manager_e2e_patch.yaml

```diff
@@ -4,3 +4,7 @@
 - op: add
   path: /spec/template/spec/containers/0/args/-
   value: --feature-gates=MultiKueueBatchJobWithManagedBy=true,TopologyAwareScheduling=true,LocalQueueMetrics=true
+- op: add
+  path: /spec/strategy
+  value: 
+    type: Recreate
```

Comment:
I don't see this needed, can you elaborate? I would just suggest to trigger the restart and wait for Kueue to be available again.

Reply:
This is no longer needed as metioned in https://github.com/kubernetes-sigs/kueue/pull/4112#discussion_r1945023638